docs/_config.yml

```
@@ -42,15 +42,15 @@ remote_theme: rundocs/jekyll-rtd-theme@v2.0.10
 #   - vendor/ruby/
 
 # Document versioning
-version: master
+version: v0.6.0
 display_version_list: true
 
 # Release is the full released version number. Used to make external links to
 # point to the correct blobs in the Github repo. This is also the version shown
 # in the sidebar (top left corner of the page)
-release: master
+release: v0.6.0
 
 # Container image which to point to in the documentation
-container_image: gcr.io/k8s-staging-nfd/node-feature-discovery-operator:master-minimal
+container_image: registry.k8s.io/nfd/node-feature-discovery-operator:v0.6.0
 # Operand Node Feature Discovery documentation version
-operand_version: master
+operand_version: v0.12.1
```

IIRC we made the same mistake last time 😎. We should probably update the comment that this should be the "release branch version" or something, I don't know how to put it

```suggestion
operand_version: v0.12
```